src/test/java/org/folio/rest/impl/LostItemFeePoliciesAPITest.java

```diff
@@ -41,207 +43,223 @@
 
 @RunWith(VertxUnitRunner.class)
 public class LostItemFeePoliciesAPITest {
```

Comment:
Indentation is not the only change in this class, is that correct?

Reply:
Yes. I brought back the original indentation.

Reply:
Thanks, it makes it much easier to review. We'll fix indentation some time later with a separate PR.